mlrun/datastore/datastore_profile.py

```diff
@@ -462,6 +485,47 @@ def from_dsn(cls, dsn: str, profile_name: str) -> "DatastoreProfileTDEngine":
         )
 
 
+class DatastoreProfilePostgreSQL(DatastoreProfile):
```

Comment:
Please add this to `test_plot_monitoring_serving_graph` and any other tests parameterized on the TSDB profile.

Reply:
will add